examples/src/bin/tlsserver-mio.rs

```diff
@@ -640,7 +640,9 @@ fn make_config(args: &Args) -> Arc<rustls::ServerConfig> {
 
     config.key_log = Arc::new(rustls::KeyLogFile::new());
 
-    if args.flag_resumption {
+    if args.flag_no_resumption {
+        config.session_storage = Arc::new(rustls::server::NoServerSessionStorage {});
+    } else {
         config.session_storage = rustls::server::ServerSessionMemoryCache::new(256);
```

Comment:
Could remove this branch, as it is the same as the default.

Reply:
This is updated now.